src/utilities/database_utils.tsx

```diff
@@ -10,13 +10,16 @@ export async function putNewDoc(
     db: PouchDB.Database<{}>,
     name: string,
     date: Date,
+    dbName: string
 ): Promise<void> {
     void db.putIfNotExists({
         _id: name,
         metadata_: {
             created_at: date,
             last_modified_at: date,
             attachments: {},
+            workflowName: dbName,
```

Comment:
We are using snake_case for all path parts, so this should be `workflow_name`. Or maybe it should be `workflow_db_name`. And it should be `project_name` below.

We should probably also include `workflow_title` from _templates_config.ts_. We can use that in the Report tab of the template.